Makefile

```diff
@@ -93,6 +96,16 @@ lint_yaml: normalize_yaml ## Lints YAML files
 lint_yarn_workspaces: ## Lints Yarn workspace packages
 	scripts/validate-workspaces.js
 
+lint_gemfile_lock: Gemfile Gemfile.lock
+	@bundle check
+	@git diff-index --quiet HEAD Gemfile.lock || (echo "Error: There are uncommitted changes after running 'bundle install'"; exit 1)
+
+lint_yarn_lock: package.json yarn.lock
```

Comment:
This should probably consider Workspace `package.json` as well, but I'd like to get this merged sooner than later to avoid developer issues on new branches from `main`. I'll look to follow-up.